exercises/practice/custom-set/.meta/solutions/CustomSet.pm

```diff
@@ -63,23 +59,12 @@ sub add {
 
 sub intersection {
     my ( $self, $other ) = @_;
-    return __PACKAGE__->new(
-        elements => [
-            grep { $self->contains($_) && $other->contains($_) } $self->_keys,
-            $other->_keys
-        ]
-    );
+    return __PACKAGE__->new( elements => [ grep { $self->contains($_) && $other->contains($_) } $self->_keys, $other->_keys ] );
 }
 
 sub difference {
     my ( $self, $other ) = @_;
-    return __PACKAGE__->new(
-        elements => [
-            grep { $self->contains($_) && !$other->contains($_) }
-                $self->_keys,
-            $other->_keys
-        ]
-    );
+    return __PACKAGE__->new( elements => [ grep { $self->contains($_) && !$other->contains($_) } $self->_keys, $other->_keys ] );
```

Comment:
This does not look like an improvement

Reply:
I can likely change this formatting manually, I'm not too concerned with example solutions but its a good point nonetheless.